<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SHRIDHARN-WIN10" ignored="0" name="Test" tests="4" failures="3" timestamp="02 Jul 2019 11:06:07 GMT" time="6.65" errors="0">
  <testcase name="basicAuthorization" time="3.734" classname="com.apiTests.BasicAuthTests"/>
  <testcase name="basicAuthorization" time="1.318" classname="com.apiTests.BasicAuthTests">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	same how bad request or credentials worng  expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
same how bad request or credentials worng  expected [200] but found [401]
at com.apiTests.BasicAuthTests.basicAuthorization(BasicAuthTests.java:160)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- basicAuthorization -->
  <testcase name="basicAuthorization" time="0.758" classname="com.apiTests.BasicAuthTests">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	same how bad request or credentials worng  expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
same how bad request or credentials worng  expected [200] but found [401]
at com.apiTests.BasicAuthTests.basicAuthorization(BasicAuthTests.java:160)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- basicAuthorization -->
  <testcase name="basicAuthorization" time="0.768" classname="com.apiTests.BasicAuthTests">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	same how bad request or credentials worng  expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
same how bad request or credentials worng  expected [200] but found [401]
at com.apiTests.BasicAuthTests.basicAuthorization(BasicAuthTests.java:160)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- basicAuthorization -->
</testsuite> <!-- Test -->
